base/special/expm1.jl

```diff
@@ -1,3 +1,4 @@
+# This code is a Julia translation of the C code from ... with the following license:
```

Comment:
the `...` should be replaced with the actual source(s) of the C code

Reply:
Ah, I actually saw that, but I was pretty sure I changed it. Lol

Reply:
Just double checked , we’re you commenting on an old commit?